src/libUtils/SWInfo.h

```diff
@@ -22,8 +22,9 @@
 #include <iostream>
 #include "common/Serializable.h"
 
-const std::string ZILLIQA_BRAND =
-    "Copyright (C) Zilliqa. Version 4.0.2. <https://www.zilliqa.com/> ";
+const std::string VERSION_TAG = "v4.0.2";
```

Comment:
Since the version place changed, maybe the release document also need to be updated.